e2e/cypress/integration/integrations/regen_token_spec.js

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @messaging
+
+import * as TIMEOUTS from '../../fixtures/timeouts';
+
+let token1;
+let token2;
+let testTeam;
+
+describe('Integrations', () => {
+    before(() => {
+        // # Login as test user and visit the newly created test channel
+        cy.apiInitSetup().then(({team}) => {
+            testTeam = team;
+            cy.visit(`/${team.name}/integrations/commands/add`);
+        });
+    });
+    after(() => {
+        // # clean up - remove slash command
+        cy.visit(`/${testTeam.name}/integrations/commands/installed`);
+        cy.get(':nth-child(3) > .color--link > span').click();
+        cy.get('#confirmModalButton').click();
+    });
+    it('MM-T581 Regen token', () => {
+        // # setup slash command
+        cy.get('#displayName').type('Token Regen Test');
+        cy.get('#description').type('test of token regeneration');
+        cy.get('#trigger').type('regen');
+        cy.get('#url').type('http://hidden-peak-21733.herokuapp.com/test_inchannel');
```

Comment:
This is fine as is but I'll update this later since we should be adding endpoint to `webhook_serve.js` and not rely to this external URL.